src/Renci.SshNet/ConnectionInfo.cs

```diff
@@ -55,7 +55,9 @@ public class ConnectionInfo : IConnectionInfoInternal
         /// <summary>
         /// Gets supported encryptions for this connection.
         /// </summary>
+#pragma warning disable CA1859 // Use concrete types when possible for improved performance
```

Comment:
We should probably delete this at some point:

https://github.com/sshnet/SSH.NET/blob/7b58819537798bec53e4193a3e5f395bcf018023/.editorconfig#L681

cc @jscarle